content/docs/studio/create-view.md

```diff
@@ -1,67 +1,71 @@
 # Create a View
 
-To create a new view for your repository, follow these steps.
+You can create views from your Git repositories, just like the `Demo` view you
+saw in the last section. To create a new view, follow these steps.
 
-1. Sign into your [DVC Studio](https://studio.iterative.ai/) dashboard.
-   ![](https://static.iterative.ai/img/studio/login_home.png)
+1. Sign into your [DVC Studio](https://studio.iterative.ai/) dashboard using
+   GitHub, GitLab or Bitbucket.
 
 2. Click on `Add a View`. All the organizations that you have access to will be
-   listed. ![](https://static.iterative.ai/img/studio/create_view.png)
+   listed.
+
+> To create views from your GitHub repositories, you must install the DVC Studio
+> GitHub app. Refer to the section on
+> [GitHub app installation](#dvc-studio-github-app-installation) for more
+> details.
 
 3. Open the organization whose repository you want to connect to. You can also
-   use the search bar to search directly for the repo which you want to connect
-   to. ![](https://static.iterative.ai/img/studio/select_repo.png)
+   use the search bar to search directly for the repository. Click on the
+   required repository to connect to it.
 
-4. Select the Git repository for which you want to create a view. For this,
-   hover over the required repository and click `Connect`.
+   ![](https://static.iterative.ai/img/studio/select_repo_v2.png)
```

Comment:
I don't think we need any of these `_v2` things

- we'd initially used `static.iterative.ai` because the images were WIP and we knew we'd change them
- now that they're finalised (I assume? #2469) we can put them in this repo as usual

CC @shcheklein related #93 <- #1115

Reply:
The reason I used _v2 is because I needed the images uploaded for testing locally. But didn't want to mess up the prod versions until the PR was reviewed and ready.

Reply:
Added to https://github.com/iterative/dvc.org/issues/2515 for removing _v2 from the image filenames (after launch).